src/main/java/org/tugraz/sysds/runtime/instructions/spark/utils/RDDConverterUtilsExt.java

```diff
@@ -103,11 +103,19 @@
 	}
 
 	public static MatrixBlock convertPy4JArrayToMB(byte [] data, long rlen, long clen) {
-		return convertPy4JArrayToMB(data, (int)rlen, (int)clen, false);
+		return convertPy4JArrayToMB(data, (int)rlen, (int)clen, false, 2);
```

Comment:
why cast? can the inputs not be Ints per default?

Reply:
I removed the long methods for now, because I agree they should never get used by py4j at the moment.